src/pages/write-for-egghead/topics.js

```diff
@@ -0,0 +1,171 @@
+import React from 'react'
```

Comment:
[suggestion: technical] these should probably be typescript (using the `tsx` and not `js` extension)

Applies to **all** components in this repository.

We currently have it set to be lenient to js files, and if it's a struggle we can discuss it. There's some value in being consistent, but there's a cost to the requirement too so it's open for discussion.

Reply:
Will do - have the basics of TS on my learning list this week. Will make updating old JS files part of that project 👍